core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -2023,4 +2023,7 @@ CacheConfigurationException storeConfiguredHasBothReadAndWriteOnly(String storeC
    @LogMessage(level = WARN)
    @Message(value = "ClusterLoader has been deprecated and will be removed in a future version with no direct replacement", id = 595)
    void warnUsingDeprecatedClusterLoader();
+
+   @Message(value = "Store %s cannot be configured to be transactional as the implementation characteristic doesn't support it", id = 596)
```

Comment:
I think this could be clearer. How about:

"Store %s cannot be configured to be transactional as it does not contain the TRANSACTIONAL characteristic" ?

Reply:
Sure.